tests/Aspire.Components.Common.Tests/Aspire.Components.Common.Tests.csproj

```diff
@@ -2,6 +2,10 @@
 
   <PropertyGroup>
     <TargetFramework>$(DefaultTargetFramework)</TargetFramework>
+    <!-- https://learn.microsoft.com/dotnet/core/testing/microsoft-testing-platform-exit-codes -->
+    <!-- Exit code 8 is "zero tests ran" -->
+    <!-- Currently, none of the tests in this project run in CI. All are ignored -->
```

Comment:
This is not a real test project. It needs `<IsTestUtilityProject>true</IsTestUtilityProject>` which got  dropped in #8279 . Adding that back should mean the project would get skipped completely.

Reply:
@radical, no, this _IS_ a test project. It contains tests. For example:
https://github.com/dotnet/aspire/blob/8489a323a5799e475f931c67f77362cb1646f47f/tests/Aspire.Components.Common.Tests/ConformanceTests.cs#L106-L109
https://github.com/dotnet/aspire/blob/8489a323a5799e475f931c67f77362cb1646f47f/tests/Aspire.Components.Common.Tests/ConformanceTests.cs#L120-L121

All non-test infra was moved to Aspire.TestUtils.

Reply:
But these tests are never run by themselves. They are run through other test classes that extend this `ConformanceTests` class.

Reply:
> But these tests are never run by themselves. They are run through other test classes that extend this `ConformanceTests` class.

In this case the project should be renamed to remove ".Tests" suffix.

Reply:
Could we do in a follow-up PR?